FirebaseAuth.podspec

```diff
@@ -12,7 +12,10 @@ supports email and password accounts, as well as several 3rd party authenticatio
   s.license          = { :type => 'Apache', :file => 'LICENSE' }
   s.authors          = 'Google, Inc.'
 
-  s.source           = { :git => 'https://github.com/firebase/firebase-ios-sdk.git', :tag => s.version.to_s }
+  s.source           = {
+    :git => 'https://github.com/firebase/firebase-ios-sdk.git',
+    :tag => s.version.to_s
```

Comment:
This `:tag` doesn't make any sense. The tags we're laying down are the Firebase release numbers (though we haven't been terribly good about that either). There is no `4.3.2` tag, and even if there were that would have been for an other Firebase-wide release number.

Shouldn't this tag be something like `auth-4.3.2` if were to exist at all?

Otherwise it seems like we should be using the Firebase version for the tag.

Reply:
Now that the open source repo is interoperable with the released Firebase version, it needs to be specified which version of Firebase, since Firebase has exact version numbers for each of the subspec dependencies.

The specified version (4.6.0 here) needs to be updated the same time all of the respective versions in each podspec is updated to the component version that goes with 4.6.0.

In the interim period, while we're still doing the official release internally and only overriding from open source, the tags are no-ops. We can't push these podspec because we don't want to replace the official internally replaced version.

There is an ongoing discussion with product management about whether we want to keep versioning the components individually. If we do so, the tags will switch to something like `"Auth-" + s.version.to_s` and we'll add to the appropriate repo. This will happen once we're ready do the official release from open source.